tests/karma/spec/domready.spec.js

```diff
@@ -0,0 +1,12 @@
+import domready from 'app/modules/domready'
+
+describe('domready', () => {
+  let count = 1
+  domready(() => { count++ }) // 2
+  domready(() => { count++ }) // 3
+  domready(() => { count++ }) // 4
+
+  it('should execute a callback given once the dom is ready', () => {
+    expect(count).to.equal(4)
+  })
+})
```

Comment:
To test that it works concurrently, ie: more than once. Several modules will add callbacks that need to be called in succession.